platform_services/components/crud/getList.js

```diff
@@ -80,11 +79,11 @@ module.exports = (query, onComplete) => {
 				attributeValues[(":" + key_name)] = {
 					'S': query[key]
 				};
-			}   
+			}
 		});
     }
 
-    if (global.userId && !_.includes(global.config.admin_users, global.userId.toLowerCase())) {
```

Comment:
This logic should reside in the crud only. The consumer of the crud can ask for all records, but if they are not the admin(s), it should just return the current logged in user records.

Reply:
Ok that makes sense. Thanks for the clarification